src/app/modules/angular-slickgrid/services/__tests__/grid-odata.service.spec.ts

```diff
@@ -1539,6 +1595,62 @@ describe('GridOdataService', () => {
         expect(query).toBe(expectation);
         expect(currentFilters).toEqual(presetFilters);
       });
+
+      it('should return a query to filter a search value with a fraction of a number that is missing a leading 0', () => {
+        const expectation = `$filter=(Duration eq 0.22)`;
+        const mockColumnDuration = { id: 'duration', field: 'duration', type: FieldType.number } as Column;
```

Comment:
please change the `type` to `type: FieldType.float`, that should increase test coverage